src/js_parser.zig

```diff
@@ -17355,9 +17356,153 @@ fn NewParser_(
         ) []Stmt {
             switch (stmtorexpr) {
                 .stmt => |stmt| {
-                    var stmts = p.allocator.alloc(Stmt, 1) catch unreachable;
-                    stmts[0] = stmt;
-                    return stmts;
+                    var class = stmt.data.s_class.class;
+                    var ctor: ?*E.Function = null;
+
+                    var hasDecorators: bool = class.ts_decorators.len > 0;
```

Comment:
The convention with zig is underscore_casing for variable names, camelCase() for functions and PascalCase for types

That being said, I think it's stupid for this to be enforced in any other way than linter and even then, I think there are cases where it's more clear to make it camelCase to match JS (such as `globalThis` since that is what it is called in JavaScript) 

This is one case where it would make more sense for it to be underscore_case, but I don't think its necessary to fix it

Reply:
👍 Switched to snake case